examples/simple_spgemm.cpp

```diff
@@ -49,15 +49,15 @@ int main(int argc, char** argv) {
 
   csr_view<T> c(nullptr, c_rowptr.data(), nullptr, {m, n}, 0);
 
-  auto info = multiply_inspect(scaled(1.f, a), b, c);
+  auto info = multiply_execute(scaled(1.f, a), b, c);
```

Comment:
Actually if I remember correctly it should be  multiply_compute()  since we had the following stages for the multi-stage cases: 

multiply_inspect ()  optional
multiply_compute()  -- first stage
multiply_fill() -- second stage